recipes/_update_packages.rb

```diff
@@ -26,3 +26,45 @@
     command "DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" upgrade && apt-get autoremove"
   end
 end
+
+# Reboot after packages update
+if !tagged?('rebooted') && node['cfncluster']['update_packages']['reboot'] == 'true'
+  # On Ubuntu, the /tmp folder is erased by default after a reboot and its lifecycle is managed
+  # by different tools, depending on the Ubuntu version
+  if node['platform'] == 'ubuntu'
+    if node['platform_version'] == "16.04"
+      file '/etc/tmpfiles.d/tmp.conf' do
+        content 'd /tmp 1777 root root 1d'
```

Comment:
What does this do?

Reply:
It prevents the /tmp folder to be erased after reboot, which is the default on Ubuntu. /tmp folder is where all the kitchen test environment is copied.
https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html